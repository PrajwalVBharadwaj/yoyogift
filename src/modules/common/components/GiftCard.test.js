import React from "react";
import { shallow } from "../../../Enzyme";
import { GiftCard } from "./GiftCard";

describe("Testing Gift card component", () => {
  it("Snapshot testing", () => {
    const wrapper = shallow(
      <GiftCard
        classes={{}}
        giftCard={{
          cardName: "Test1",
          cardEmail: "test@e.com",
          id: 1,
          cardPoints: 2000,
          cardImage: "link"
        }}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
