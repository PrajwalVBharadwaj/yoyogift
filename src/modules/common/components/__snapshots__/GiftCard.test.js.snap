// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Gift card component Snapshot testing 1`] = `ShallowWrapper {}`;
